src/routes/index.js

```diff
@@ -12,7 +12,7 @@ router.get('/', function(req, res) {
 
     res.redirect('/' + deviceId);
   } else {
-    res.render('error', {});
+    res.render('device-view', {});
```

Comment:
Returned an empty error page when no devices were found. Now shows the page again.